examples/time_series_forecasting.ipynb

```diff
@@ -0,0 +1,472 @@
+{
```

Comment:
<p>Typo: "additoinal" should be "additional"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/giotto-ai/giotto-tda/pull/475/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
Fixed, thanks.